tools/dataset_converters/sunrgbd_data_utils.py

```diff
@@ -8,28 +8,24 @@
 from scipy import io as sio
 
 
-def random_sampling(points, num_points, replace=None, return_choices=False):
```

Comment:
Is it safe if we directly delete the argument of return_choices?
If it is not harmful, can we consider keeping return_choices so that the functionality is only enhanced by

Reply:
Wenwei's comment means that we should keep the original function, including the argument `return_choices`. We should keep the original behavior to make it consistent with master.